vars/notify.groovy

```diff
@@ -388,11 +388,40 @@ def log(message, args=[:]) {
                           "--severity=${level} " +
                           logName + " " +
                           exec.shellEscape(jsonMessage));
+
+      def currentResult = currentBuild.result
+
       sh(shellCommand);
```

Comment:
`shellCommand` should totally be a list, and we should be using exec() here.  Not that it would help in itself, but then we could put this check in exec.groovy rather than here, and have it apply in more places.

Reply:
Definitely!  The idea is to confirm that's what we need to do and then work my way thru all the spots the call `sh` and replace them with something that can detect aborted states like I added here.

I'd hate to have this show up again thinking we fixed it :(